srcapi/store/config.ts

```diff
@@ -1382,12 +1382,20 @@ export type gmfCustomCSS = {
  */
 export type pytreeLidarprofileJsonUrl = string;
 
+export type ogcServerDatasourceOptions = {
+  /**
+   * The WMS info format to be used with Arcgis OGC server.
+   */
+  arcgisWMSInfoFormat: string;
+};
 export type gmfDatasourceOptions = {
   /**
    * The WMS info format to be used with Arcgis OGC server.
    */
   arcgisWMSInfoFormat: string;
+  ogcServerSpecific: {[x: string]: ogcServerDatasourceOptions};
```

Comment:
just to be sure, is it normal that one is ogcServerSpecific (Server without "s") while the property name in src/datasource/Manager.js is ogcServersSpecific (Servers with a "s") ?

Reply:
Effectively thanks, fixed :-)